manager/integration/tests/test_upgrade.py

```diff
@@ -133,22 +135,26 @@ def test_upgrade(longhorn_upgrade_type,
 
     1. Find the upgrade image tag
     2. Create a volume, generate and write data into the volume.
-        1. Create a volume with revision counter enabled case.
-        2. Create a volume with revision counter disabled case.
+        1. Create volume0 with revision counter enabled case.
```

Comment:
Since we cannot split test cases for the upgrade, maybe we can have a meaningful volume name. ex `volume_revision_enabled`, `volume_revision_disabled`, `volume_rebuild`?

Reply:
Hi @c3y1huang :
     I update the PR. please review it again.